internal/internal_task_handlers.go

```diff
@@ -115,8 +115,10 @@ type (
 		isWorkflowCompleted bool
 		result              *commonpb.Payloads
 		err                 error
-
+		// previousStartedEventID is the event ID of the workflow task started of the previous workflow task.
```

Comment:
```suggestion
		// previousStartedEventID is the ID of the workflow task started event of the previous workflow task.
```

Or also keep "event" before ID if you want